src/prevector.h

```diff
@@ -394,6 +394,20 @@ class prevector {
         fill(ptr, first, last);
     }
 
+    inline void resize_uninitialized(size_type new_size) {
+        // resize_uninitialized change the size of the prevector but does not initialize.
+        // If size < new_size, the added elements must be initialized explicitly after it returns.
+        difference_type count = new_size - size();
```

Comment:
Is this always guaranteed to give a signed result back? I.e. if `new_size` is 4 and `size()` is 5, will `count` always be `-1` or will it sometimes be cast to `(uint32_t)-1`?

Reply:
@kallewoof Thanks. 
I will judge whether the value of '_size' should need to increase by comparing new_size and size () instead of the sign of pre-computed difference of them.

Reply:
done.